doc/classes/AudioEffectLowShelfFilter.xml

```diff
@@ -1,9 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioEffectLowShelfFilter" inherits="AudioEffectFilter" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Reduces all frequencies below the [member AudioEffectFilter.cutoff_hz].
+		Adds a low-shelf filter to the Audio bus.
 	</brief_description>
 	<description>
+		Reduces all frequencies below the [member AudioEffectFilter.cutoff_hz].
```

Comment:
Is this a high-pass or a low-pass filter? Looking at the description, this is a high-pass filter, but I think the description should mention it. It seems like it's the opposite of shelf (i.e. high-pass is low-shelf)?

Reply:
After 5 minutes of googling (I'm not an audio person) yes it looks like they're the opposite of a shelf. I wouldn't call it a high-pass filter though. Shelf filters only reduce frequencies, pass filters cut them off entirely.